src/pages/main/ui/MainPage.async.tsx

```diff
@@ -0,0 +1,6 @@
+import { lazy } from 'react';
+
+export const MainPageAsync = lazy(() => new Promise(resolve => {
+    // @ts-ignore
+    setTimeout(() => resolve(import('./MainPage')), 1500)
```

Comment:
давай уменьшим до 500? сильно уж долго грузится
я про все места такие имею в виду
```suggestion
    setTimeout(() => resolve(import('./MainPage')), 500)
```